pom.xml

```diff
@@ -87,6 +87,17 @@
       <version>0.1.1-SNAPSHOT</version>
     </dependency>
 
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-data-mongodb</artifactId>
+    </dependency>
+    <!-- https://mvnrepository.com/artifact/org.mongodb/mongodb-crypt -->
+    <dependency>
+      <groupId>org.mongodb</groupId>
+      <artifactId>mongodb-crypt</artifactId>
+      <version>1.4.0</version>
+    </dependency>
```

Comment:
Can `mongodb-crypt` be removed for now?

Reply:
@jkotanchik-SB Yeah it should be removed. I just did. Thanks!